applications/Parallel.md

```diff
@@ -129,38 +129,38 @@ The lending protocol was inspired by compound protocol and our blockchain soluti
-|    1.a | Substrate module: XCMP     | We will create a Substrate module that will utilize XCMP protocol to enable cross-chain transfer of DOT and KSM through Relay chain                                                                                     |
-|    1.b | Testing and Documentation  | While we develop our module, we will ensure that our code has significant test coverage (>80%) to ensure quality and provide explanations on how the functions work for the community to review, as well as craft test guides |
-|    2.a | Substrate module: Price aggregation oracle   |  We will need to create a on-chain price aggregation pallet for the oracle prices from multiple source that is queried by off-chain workers.                                                                            |
-|    2.b | Testing and Documentation  | While we develop our module, we will ensure that our code has significant test coverage (>80%) to ensure quality and provide explanations on how the functions work for the community to review, as well as craft test guides |
-|    3.a | Integration with front-end | We will integrate our existing front end to the finalized substrate backend                                                                                                                                                   |
-|    3.b | Article/Tutorial           | We will create an article and a video demo that explains how users can start using the platform in lending and borrowing for DOT and KSM                                                                                      |
-|    3.c | Docker                     | We will provide a dockerfile to demonstrate the full cross-chain functionality                                                                                                                                                |
-|     4. | User Testing               | We will conduct user testing to improve our product's UX and UI, to ensure that the borrowing and lending functionalities are intuitive. Initially, we will conduct qualitative user testing by observing 10-15 users use the v1 platform and provide a summary of the findings and improvements made based on insights.                                                                            |
+|    1.a | Substrate module: Loans pallet | Loans pallet will be implemented as a multi-assets lending protocol which offers lending and borrowing by using floating rate model.                                                                                     |
+|    1.b | Substrate module: Liquidation pallet | We will implement a liquidation solution build with Substrate Off-chain Worker, which will calculate the health factor of each borrower's account and send a liquidation transaction automatically.                                                                                     |
```

Comment:
For example here

```suggestion
|    1.b | Substrate module: Liquidation pallet | We will implement a liquidation solution build with Substrate Off-chain Worker, which will calculate the health factor of each borrower's account and send a liquidation transaction automatically. The full expected functionality is described [here](https://docs.parallel.fi/white-paper#4-liquidation). |
```

assuming this pallet will indeed provide all functionality described in subsections 4.1, 4.2 and 4.3.